flutter/lib/src/web/sentry_web.dart

```diff
@@ -57,9 +60,36 @@ class SentryWeb with SentryNativeSafeInvoker implements SentryNativeBinding {
   @override
   FutureOr<void> captureEnvelope(
       Uint8List envelopeData, bool containsUnhandledException) {
-    _logNotSupported('capture envelope');
+    _logNotSupported('capture raw envelope data');
   }
 
+  @override
+  FutureOr<void> captureStructuredEnvelope(SentryEnvelope envelope) =>
+      tryCatchAsync('captureEnvelopeObject', () async {
```

Comment:
Th string here still says `captureEnvelopeObject` instead of `captureStructuredEnvelope`.